rmgpy/rmg/input.py

```diff
@@ -770,6 +775,44 @@ def uncertainty(localAnalysis=False, globalAnalysis=False, uncorrelated=True, co
     }
 
 
+def restartFromSeed(path=None, coreSeed=None, edgeSeed=None, filters=None, speciesMap=None):
+    parentDir = os.path.dirname(rmg.inputFile)
+    rmg.restart = True
+
+    if path:
+        if any((coreSeed, edgeSeed, filters, speciesMap)):
+            raise InputError('For restarting an RMG job from a seed mechanism, either the path to the RMG generated '
+                             'seed mechanism should be given as `path`, or the path for each of the required files '
+                             'should be explicitly given, but not both. Please take one approach or the other. For '
+                             'further information see the RMG documentation on restarting from a seed mechanism at'
+                             'http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/input.html#restarting-from-a-seed-mechanism.')
+
+        if not os.path.isabs(path):
+            path = os.path.join(parentDir, path)
+
+        if not os.path.exists(path):
+            raise ValueError('Unable to find the path to the restart seed folder. {0} does not exist'.format(path))
+
+        # Try to find the paths for all of the required modules
+        rmg.coreSeedPath = os.path.join(path, 'seed')
+        rmg.edgeSeedPath = os.path.join(path, 'seed_edge')
+        rmg.filtersPath = os.path.join(path, 'filters', 'filters.h5')
+        rmg.speciesMapPath = os.path.join(path, 'filters', 'species_map.yml')
+
+        if any([not os.path.exists(file_path) for file_path in [rmg.coreSeedPath, rmg.edgeSeedPath,
+                                                                rmg.filtersPath, rmg.speciesMapPath]]
+               ):
```

Comment:
I think we should do this check after the if statement, in case the use doesn't provide all 4 arguments.